selfdrive/car/hyundai/values.py

```diff
@@ -520,8 +520,9 @@ class CAR(Platforms):
   )
   GENESIS_GV70_1ST_GEN = HyundaiCanFDPlatformConfig(
     [
-      HyundaiCarDocs("Genesis GV70 (2.5T Trim) 2022-23", "All", car_parts=CarParts.common([CarHarness.hyundai_l])),
-      HyundaiCarDocs("Genesis GV70 (3.5T Trim) 2022-23", "All", car_parts=CarParts.common([CarHarness.hyundai_m])),
+      # TODO: Hyundai P is likely the correct harness for HDA II for 2.5T (unsupported due to missing ADAS ECU, is that the radar?)
+      HyundaiCarDocs("Genesis GV70 (2.5T Trim, without HDA II) 2022-23", "All", car_parts=CarParts.common([CarHarness.hyundai_l])),
+      HyundaiCarDocs("Genesis GV70 (3.5T Trim, without HDA II) 2022-23", "All", car_parts=CarParts.common([CarHarness.hyundai_m])),
```

Comment:
I don't think we make the HDA2 version of the 3.5T connector yet, user will need to re-pin

Reply:
So is it the wrong harness that was shipped? The websites recommendation was to purchase connector L as it’s recommended for the 2022 GV70 2.5T. Being that it’s the sport prestige package, should the M connector work or the P connector?
If that is the case, how do I exchange for the correct one?
If a re-pin is needed and I need to utilize a different fork that supports it (SP/FP), are there instruction as to how to re-pin?

Reply:
> So is it the wrong harness that was shipped? The websites recommendation was to purchase connector L as it’s recommended for the 2022 GV70 2.5T. Being that it’s the sport prestige package, should the M connector work or the P connector? If that is the case, how do I exchange for the correct one? If a re-pin is needed and I need to utilize a different fork that supports it (SP/FP), are there instruction as to how to re-pin?

It seems that you’ve missed the message I pinged you with on Discord. Instructions have been mentioned in my reply on Discord.

Reply:
I combination of both, likely wrong harness and there are differences with how the ECUs are structured that openpilot doesn't yet support, but may come in a software update.

Basically, we expect your car to have an ADAS ECU sending the lane following messages, but our firmware query returns nothing for the expected address.

You can try to ask for a return of the harness, but you will still be on your own for software support. @sunnyhaibin has a PR for a similar car, so this may work: https://github.com/commaai/openpilot/pull/31704